fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -3527,6 +3529,10 @@ public List<ResultRow> executeInternalQuery() {
         UUID uuid = UUID.randomUUID();
         TUniqueId queryId = new TUniqueId(uuid.getMostSignificantBits(), uuid.getLeastSignificantBits());
         context.setQueryId(queryId);
+        String stmt = SqlDialectHelper.convertSqlByDialect(originStmt.originStmt, context.getSessionVariable());
```

Comment:
internal sql is always written with Doris' dialect, so it should not convert by dialect?

Reply:
removed